cobc/ChangeLog

```diff
@@ -1,4 +1,12 @@
 
+2023-10-02  Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>
+
+	* error.c (print_error_prefix), flag.def: new flag
+	  -fdiagnostics-absolute-paths to print the full path of
+	  a file for diagnostics; this flag can be activated if
+	  your editor and build system do not correctly work
+	  together to locate files from diagnostic output
```

Comment:
should we move this longer description to the NEWS and keep the reasoning out here (or have it in both)?

Reply:
ping (NEWS entry would be necessary in any case)